aopy/data/db.py

```diff
@@ -414,6 +443,96 @@ def get_db_object(self):
         '''
         return self.record
 
+class BMI3DDecoder():
+    '''
+    Wrapper for BMI3D Decoder objects
+    '''
+    
```

Comment:
Can you add a little more information about what this class does with BMI3D decoder objects?

Reply:
it's a wrapper :)